test/integration/database.test.js

```diff
@@ -5,16 +5,19 @@
  */
 
 /*
- * Copyright (c) 2014, Joyent, Inc.
+ * Copyright (c) 2020, Joyent, Inc.
  */
 
 var vasync = require('vasync');
 
-var core = require('../lib');
+var core = require('../../lib');
+
+var tap = require('tap');
+
 
 if (require.cache[__dirname + '/helper.js'])
         delete require.cache[__dirname + '/helper.js'];
```

Comment:
Would you consider ditching this `require.cache` mucking? It is gross under-the-hood node hacking to ensure that `helper.js` is *re*-loaded for each test module. That was necessary when using nodeunit that ran all the test files in a single process. Node-tap runs each test file in a separate process so it should no longer be necessary.

Reply:
Done.